gym_xiangqi/envs/xiangqi_env.py

```diff
@@ -95,6 +98,16 @@ def __init__(self, agent_color=RED):
         else:
             self.enemy_color = RED
 
+        # track whose turn it is
+        if self.agent_color == RED:
+            self.turn = AGENT
+        else:
+            self.turn = ENEMY
+
+        # epoch termination flag
+        self._done = False
+        self.done_warn = False
```

Comment:
Oh what is done_warn for again?

Reply:
This is used when printing a warning message for the users to call reset() method once the `done` flag is `True`. Instead of endlessly printing the warning message `done_warn` will only print the message once per epoch. 
I don't think these are super necessary features. We can remove if they seem too trivial. I saw some examples of other environment doing this so tried putting them for now.